l10n-pocket/en/footer.ftl

```diff
@@ -35,3 +36,24 @@ pocket-footer-cookie-preferences = Cookie preferences
 pocket-footer-read-it-later-brand-name = { -brand-name-read-it-later-inc }
 pocket-footer-twitter-brand-name = { -brand-name-twitter }
 pocket-footer-facebook-brand-name = { -brand-name-facebook }
+
+### Platform footer is used in some pages:
```

Comment:
This should be a group level comment (Tripplt hash comments apply to the whole file)

Reply:
Should also remove the blank line below here if this belongs to the same multi-line comment underneath?

Reply:
Makes sense!
And is it fine to have multiple group-level (double dash) comments following each other? Such as:
```## Platform navigation is used in some pages:
## Pocket homepage https://www-dev-pocket.allizom.org
## Add page URL https://www-dev-pocket.allizom.org/add
## All platform pages
## https://www-dev-pocket.allizom.org/ios
## https://www-dev-pocket.allizom.org/android
## https://www-dev-pocket.allizom.org/chrome
## https://www-dev-pocket.allizom.org/safari
## https://www-dev-pocket.allizom.org/opera
## https://www-dev-pocket.allizom.org/edge
```
I think this is the part that mainly had me feeling a little lost with how to properly utilize Fluent comments